recipe/update-activate-and-repack.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+set -exuo pipefail
+
+src="${SRC_DIR}/${PKG_NAME}"
+
+CBUILD=$(${PREFIX}/bin/icx -dumpmachine | sed "s/unknown/conda/")
+CHOST=$(${PREFIX}/bin/icx -dumpmachine | sed "s/unknown/conda/")
+
+FINAL_CFLAGS="-march=nocona -mtune=haswell -ftree-vectorize -fPIC -fstack-protector-strong -fno-plt -O2 -ffunction-sections -pipe"
+FINAL_DEBUG_CFLAGS="-march=nocona -mtune=haswell -ftree-vectorize -fPIC -fstack-protector-all -fno-plt -Og -g -Wall -Wextra -fvar-tracking-assignments -ffunction-sections -pipe"
+FINAL_CPPFLAGS="-DNDEBUG -D_FORTIFY_SOURCE=2 -O2 -fuse-ld=lld"
+FINAL_DEBUG_CPPFLAGS="-D_DEBUG -D_FORTIFY_SOURCE=2 -Og -fuse-ld=lld"
+FINAL_CXXFLAGS="-fvisibility-inlines-hidden -std=c++17 -fmessage-length=0 -march=nocona -mtune=haswell -ftree-vectorize -fPIC -fstack-protector-strong -fno-plt -O2 -ffunction-sections -pipe"
+FINAL_DEBUG_CXXFLAGS="-fvisibility-inlines-hidden -std=c++17 -fmessage-length=0 -march=nocona -mtune=haswell -ftree-vectorize -fPIC -fstack-protector-all -fno-plt -Og -g -Wall -Wextra -fvar-tracking-assignments -ffunction-sections -pipe"
+FINAL_LDFLAGS="-Wl,-O2 -Wl,--sort-common -Wl,--as-needed -Wl,-z,relro -Wl,-z,now -Wl,--disable-new-dtags -Wl,--gc-sections -fuse-ld=lld"
+FINAL_LDFLAGS_LD="-O2 --sort-common --as-needed -z relro -z now --disable-new-dtags --gc-sections -fuse-ld=lld"
```

Comment:
@AndresGuzman-Ballen These are the initial flags we used to test compile packages. Would you and folks at Intel be able to review these and let us know what you would like them to be set to?

Primarily the `tune` and `arch`, as they were pulled from the GCC activation package as is.

Reply:
Let me see! Right now the flags we use are:
```
# Activation flags
FINAL_CPPFLAGS:
  - -DNDEBUG -D_FORTIFY_SOURCE=2
FINAL_CFLAGS:
  - -Wformat -Wformat-security -O3 -fp-model strict -fomit-frame-pointer -xSSE4.2 -axCORE-AVX2,COMMON-AVX512 -fPIC -fstack-protector-all
FINAL_CXXFLAGS:
  - -Wformat -Wformat-security -O3 -fp-model strict -fomit-frame-pointer -xSSE4.2 -axCORE-AVX2,COMMON-AVX512 -fPIC -fstack-protector-all
FINAL_FFLAGS:
  - -fopenmp -O3 -fp-model strict -fomit-frame-pointer -xSSE4.2 -axCORE-AVX2,COMMON-AVX512 -fPIC -fstack-protector-all
FINAL_LDFLAGS:
  - -Wl,-O3 -Wl,--sort-common -Wl,--as-needed -Wl,-z,noexecstack -Wl,-z,relro -Wl,-z,now -Wl,--disable-new-dtags -Wl,--gc-sections
FINAL_DEBUG_CPPFLAGS:
  - -D_DEBUG -D_FORTIFY_SOURCE=2 -Og
FINAL_DEBUG_CFLAGS:
  - -Wformat -Wformat-security -O0g -g -fp-model strict -fomit-frame-pointer -xSSE4.2 -axCORE-AVX2,COMMON-AVX512 -fPIC -fstack-protector-all
FINAL_DEBUG_CXXFLAGS:
  - -Wformat -Wformat-security -O0g -g -fp-model strict -fomit-frame-pointer -xSSE4.2 -axCORE-AVX2,COMMON-AVX512 -fPIC -fstack-protector-all
FINAL_DEBUG_FFLAGS:
  - -fopenmp -O0g -g -fp-model strict -fomit-frame-pointer -xSSE4.2 -axCORE-AVX2,COMMON-AVX512 -fPIC -fstack-protector-all
FINAL_DEBUG_LDFLAGS:
  - -Wl,-O0g -Wl,-g -Wl,--sort-common -Wl,--as-needed -Wl,-z,noexecstack -Wl,-z,relro -Wl,-z,now -Wl,--disable-new-dtags -Wl,--gc-sections
```

I'll ask another engineer what his thoughts are on this, so we can have this PR include just the essentials.

Reply:
@adipietro18 The flags we use should be good, specifically "-xSSE4.2 -axCORE-AVX2,COMMON-AVX512". Binary size of extension will increase the longer the -ax list of ISAs is, so default should not be too long and should include ISAs which have sufficiently penetrated the market. Compiling NumPy with the new flags you add will also be a great way for testing the flags that you add.